src/components/App.vue

```diff
@@ -26,6 +28,7 @@ interface AppBindings extends Vue {
 @Component( {
 	components: {
 		TermBox,
+		InMoreLanguageSwitch,
```

Comment:
Just to be completely explicit. We are building a button with the "Label for the button to show all non-empty sets of terms". So it should be called `In**All**LanguageSwitch` then. Not an awesome name either but depends how we organize its content (incl. hidden element, ...).

Reply:
changed